src/coreml.cs

```diff
@@ -266,5 +276,46 @@ interface MLMultiArray {
 		[Export ("setObject:forKeyedSubscript:")]
 		void SetObject (NSNumber obj, NSNumber [] key);
 	}
+
+	[Watch (4,0), TV (11,0), Mac (10,13, onlyOn64: true), iOS (11,0)]
+	[BaseType (typeof (NSObject))]
+	interface MLDictionaryConstraint {
```

Comment:
I don't see this type exposed anywhere in the PR !?! but it's used as a property in the diff. Something is missing ?